source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -715,6 +715,15 @@ namespace
                 else
                 {
                     destPort = destNodeDef->addInput(attr->getName(), attr->getType().str());
+                    const RtTypedValue* uiFolderMetaData = input->getMetadata(RtToken(ValueElement::UI_FOLDER_ATTRIBUTE));
```

Comment:
Write out the nodedef input uifolders

Reply:
Should we not copy over all ui information in this case. Not sure why just ui folder?

Reply:
Hmm.. exporting all metadata should already be handled by the call to writeMetadata above (line 701). Is that not working?